src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private final int radius;
+
+    public Circle(Color color, int radius) {
+        super(color);
+        this.radius = radius;
+    }
+
+    @Override
+    public double calculateArea() {
+        return (Math.pow(radius, 2) * Math.PI);
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: Circle"
+                + ", area: "
+                + calculateArea() + " sq.units, radius: "
+                + radius + " units, color: "
+                + getColor().toString().toLowerCase());
```

Comment:
[you need to use method `name()` to get a string representation of the enum. ](https://mate-academy.github.io/jv-program-common-mistakes/java-core/abstract-class-interface/oop-advanced#use-name-for-getting-string-representation-of-enum-constants)